cas-server-support-jpa-ticket-registry/src/main/java/org/jasig/cas/ticket/registry/JpaTicketRegistry.java

```diff
@@ -168,8 +171,10 @@ public Ticket getTicket(final String ticketId) {
      */
     private Ticket getRawTicket(final String ticketId) {
         try {
-            if (ticketId.startsWith(this.ticketGrantingTicketPrefix)) {
-                return entityManager.find(TicketGrantingTicketImpl.class, ticketId);
+            for(final String prefix : this.ticketGrantingTicketPrefixes) {
+                if (ticketId.startsWith(prefix)) {
+                    return entityManager.find(TicketGrantingTicketImpl.class, ticketId);
```

Comment:
Shouldn't this also account for ProxyGrantingTicketImpl? I realize they are both the same entity, but for the sake of clarity, it might be best to include them both, or at the least, explain this via javadocs.